commit c38311a4e29cdb61735a3d1700e1d1142cb080f3
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-09-16T17:22:20+00:00 (3 weeks ago)
    
    TOOLS-2319 eng.git never passes the gerrit bot 'make check'
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 523eaef..e0b5eef 100644
--- a/Makefile
+++ b/Makefile
@@ -129,6 +129,8 @@ include ./deps/eng/tools/mk/Makefile.smf.defs
 NODE_PREBUILT_VERSION =	v6.17.0
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG = zone64
+	# Use the sdcnode build for minimal-64-lts@18.4.0
+	NODE_PREBUILT_IMAGE = c2c31b00-1d60-11e9-9a77-ff9f06554b0f
 	include ./deps/eng/tools/mk/Makefile.node_prebuilt.defs
 else
 	NPM=npm
diff --git a/tools/mk/Makefile.targ b/tools/mk/Makefile.targ
index 755107f..424a53b 100644
--- a/tools/mk/Makefile.targ
+++ b/tools/mk/Makefile.targ
@@ -235,8 +235,11 @@ deps/%/.git:
 # would invoke bash with zero files), and you can check individual files from
 # the command line with "make filename.bashchk".
 #
+# Since bashstyle is a node program, we need our node runtime as a dependency
+# for running $(BASHSTYLE)
+#
 .PHONY: check-bash
-check-bash: $(BASH_FILES:%=%.bashchk) $(BASH_FILES:%=%.bashstyle)
+check-bash: $(NODE_EXEC) $(BASH_FILES:%=%.bashchk) $(BASH_FILES:%=%.bashstyle)
 
 %.bashchk: %
 	$(BASH) -n $^
